app/services/clinic_finder/modules/geocoder.rb

```diff
@@ -6,49 +6,17 @@ module Modules
     # Nesting due to rails autoload
     module Geocoder
       # Get a patient's latitude and longitude.
-      def get_patient_coordinates_from_zip(zipcode)
+      def patient_coordinates_from_zip(zipcode)
         patient_location = @geocoder.geocode zipcode
-        @patient_context.location = patient_location.ll
+        patient_location.ll
```

Comment:
I understand it might be out of scope - but I really hate `.ll` - its really short and its made out of indistinct letters. Could we maybe change it? :grinning:

Reply:
haha I hear you. It looks like this works if instead of `ll` we do `[location.lat, location.lng]` which is a little more verbose but seems to blend okay. I'll swap that out where it's used.